pocs/mount/mount.py

```diff
@@ -292,6 +298,20 @@ def get_current_coordinates(self):
 
         return self._current_coordinates
 
+    def distance_from_target(self):
```

Comment:
This is only used by Huntsman so far but it doesn't hurt to have here. Could be convenient.

Reply:
If you have more than one camera, what does it mean?

Reply:
In the case of Huntsman all the cameras are (roughly) pointing in the same direction so all have a common distance to target.

Reply:
More generally, as a method of the mount this would refer to the distance on the sky between the target coordinates and the 'mount coordinates', which will always be unique even in the case where multiple cameras are attached to a single mount.

In practice the mount coordinates would be defined by selecting one camera as the reference, and using that camera to do the pointing tests that are used to calibrate the mount pointing model and thereby calculate mount position.